src/test/java/bio/terra/flight/study/create/StudyCreateFlightTest.java

```diff
@@ -0,0 +1,98 @@
+package bio.terra.flight.study.create;
+
+import bio.terra.category.Connected;
+import bio.terra.metadata.Study;
+import bio.terra.model.*;
```

Comment:
another nit *. i think if you do "optimize imports" with the java format we are using it will replace * with the actual classes. see below also.